bridge/github/datakit_github.ml

```diff
@@ -95,6 +95,7 @@ module Commit = struct
 
   type t = { repo: Repo.t; id : string }
 
+  let create repo id = {repo; id = String.trim id }
```

Comment:
We could also run this in a future patch in an "invariants" mode where it issues a warning with the original callsite if `id <> String.trim id`. For now, this makes the interface more defensive, as @samoth notes